index.html

```diff
@@ -32,9 +37,27 @@
+       <h1>What is a Hackathon?</h1>
+        <p>A hack is a way to cut with rough or heavy blows. In this same sense, a hackathon involves <br> finding a rough solution to an idea over a short period of time. The lack of perfection helps <br> people experiment with ideas and tools they might not have otherwise in a more restricted environment.
+        </p>
+        <p>During the event, people come together to problem-solve or work on a particular task. <br> They typically often involve computers; however, programming is not necessary to participate since <br> they involve people in various different areas of expertise in a team to solve a problem. 
+        </p>
+        <p>Some hackathons are competitive in nature and involve prize money or job offers. <br> Others are for charity and help non-profit and not-for-profit groups achieve a goal involving a social issue. <br> They can even be coordinated by a group of friends to build a tool around personal interests! 
+        </p>
+        <h2>What to Expect at a Hackathon</h2>
+        <p>Hackathons generally last anywhere from from a night to a weekend. <br> 
+        They can also include food and social events! Here’s the general layout of what you can expect:
+        </p>
+        <ol>
+            <li>Registration</li>
+            <li>Project pitches</li>
+            <li>Introductions</li>
+            <li>Networking and formation of teams</li>
+            <li>Work</li>
+            <li>Presentation of projects</li>
+            <div>
```

Comment:
It looks like this div with an img should be outside of the ordered list. Can you do this, and indent the img element within the div?